src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -0,0 +1,348 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeCleanup;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.RemoveUnusedVariable;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Editor.CodeCleanup;
+using Microsoft.VisualStudio.Language.CodeCleanUp;
+using Microsoft.VisualStudio.LanguageServices.Implementation.CodeCleanup;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.LanguageService
+{
+    [Export(typeof(CodeCleanUpFixer))]
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    internal class CSharpCodeCleanUpFixer : CodeCleanUpFixer
+    {
+        private const string RemoveUnusedImportsFixId = "RemoveUnusedImportsFixId";
+        private const string SortImportsFixId = "SortImportsFixId";
+
+        [Export]
+        [FixId(IDEDiagnosticIds.UseImplicitTypeDiagnosticId)]
```

Comment:
I would extract these out to its own thing like http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Classification/ClassificationTypeDefinitions.cs,11

Reply:
I'm leaving this for now since we have active plans to gut this after preview 1: https://github.com/dotnet/roslyn/pull/30039#discussion_r226546564